api_factory/generator/templates/$namespace/$name_$version/$service/client.py.j2

```diff
@@ -86,8 +113,25 @@ class {{ service.name }}:
             client_info=self.client_info,
         )
 
+        {%- if method.lro_payload %}
```

Comment:
I would like to restructure this section so that the rpc is called in a non-branching path:

```python
response = rpc(
    request,
    retry=retry,
    timeout=timeout,
    metadata=metadata,
)
{% if method.lro_payload) %}
# Wrap the response in a future that knows how to handle operations.
response = operation.from_gapic(response, self._operations_client, ...)
{% endif %}

# Done; return the response
return response
```

My assertion is that this is a simpler flow, and if we have additional cases where we wrap or enhance a response object somehow, it will be easier to chain them.

One argument _against_ this precise behavior is that it changes the type of `response` somewhat willy-nilly. A compromise position for that is to still do `return operation.from_gapic(...)` directly in that if block, and shift the `return response` into an else.